Documentation/Troubleshooting/Index.rst

```diff
@@ -45,6 +45,11 @@ Use this new administrator to log into the TYPO3 backend. In the module
 :guilabel:`Backend Users` you can change the passwords of existing users,
 including admins.
 
+Alternative:
+Use a database tool as phpMyAdmin and the following SQL to set a new password::
+
+   UPDATE be_users SET password=md5('login&go') WHERE username='admin';
```

Comment:
This does not work since TYPO3 does not use/support MD5 anymore. You need to generate a proper hash using the algorithm of your instance (Argon2i, etc.) instead. And this is not something you can do in SQL.

Reply:
how can I know which algorithm is used on my TYPO3 website?

Reply:
See https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/ApiOverview/PasswordHashing/Index.html